editor/src/components/editor/store/store-deep-equality-instances.ts

```diff
@@ -2988,6 +2992,21 @@ export const UtopiaVSCodeConfigKeepDeepEquality: KeepDeepEqualityCall<UtopiaVSCo
     },
   )
 
+export const GithubRepoKeepDeepEquality: KeepDeepEqualityCall<GithubRepo> = combine2EqualityCalls(
+  (repo) => repo.owner,
+  createCallWithTripleEquals(),
```

Comment:
not strictly related to the PR but I wonder if we should make dedicated StringKeepDeepEquality and NumberKeepDeepEquality functions, because now if we change repo.owner to, say, an object, this will not break here and we will then run === equality on objects, which will always fail, and then that will bubble up to always fail the editor state deep equality too.

Reply:
Or at least pass an explicit type to that function.

Reply:
Very very good shout!